masq/src/commands/setup_command.rs

```diff
@@ -485,4 +485,37 @@ NAME                          VALUE
 
         assert_eq!(handle.stdout_so_far(), expected);
     }
+
+    #[test]
+    fn process_setup_command_for_exit_location() {
```

Comment:
You said this whole test should go away. Hopefully that means the code it's driving should go away too.